modules/010-prometheus-crd/crds/customalertmanager.yaml

```diff
@@ -838,6 +838,206 @@ spec:
                                   type: string
                               type: object
                             type: array
+                          telegramConfigs:
```

Comment:
Please describe the changes in the PR description @dmvolod

Reply:
donw